app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -200,6 +237,33 @@ public void onClick(View v) {
                 new IntentFilter(ACTION_RESET_SCREENSHOT));
         LocalBroadcastManager.getInstance(this).registerReceiver(showUpdateDialog,
                 new IntentFilter(ACTION_SHOW_UPDATE_DIALOG));
+
+        initiateTeamPickerSpinner();
+    }
+
+    /**
+     * initiates the team picker spinner
+     */
+    private void initiateTeamPickerSpinner() {
+        Spinner spinner = (Spinner) findViewById(R.id.teamPickerSpinner);
+        ArrayAdapter<CharSequence> adapter = ArrayAdapter.createFromResource(this,
+                R.array.teams, android.R.layout.simple_spinner_item);
+        adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+        spinner.setAdapter(adapter);
+
+
+        spinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parentView, View selectedItemView, int position, long id) {
+                settings.setPlayerTeam(position);
+            }
+
+            @Override
+            public void onNothingSelected(AdapterView<?> parentView) {
+            }
```

Comment:
onNothingSelected is a method which is called if the spinner is removed by any action other than the user picking an item from the list. The interface for the adapter listener requires onNothingSelected to be implemented, but we don't want anything to happen if the user dismisses the spinner or if it disappears, so I left the method empty. I can add a comment in the method saying "//We dont want anything to happen when user dismiss spinner, but interface requires implementation" if you think it'd help with clarity.